gluten-substrait/src/main/scala/org/apache/gluten/execution/ExpandExecTransformer.scala

```diff
@@ -112,13 +115,12 @@ case class ExpandExecTransformer(
 
   override protected def doTransform(context: SubstraitContext): TransformContext = {
     val childCtx = child.asInstanceOf[TransformSupport].transform(context)
-    val operatorId = context.nextOperatorId(this.nodeName)
-    if (projections == null || projections.isEmpty) {
+    if (metricsUpdater == MetricsUpdater.None) {
```

Comment:
Hi @zml1206

Can we create a new utility method, e.g, `isNoop` that can be called both by `doTransform` and `metricUpdater` in each operator? Thanks.